endpoints/users.py

```diff
@@ -0,0 +1,59 @@
+from utils.attacker import Attacker
+from utils.util import print_header
+
+
+class Users:
+    """
+    Contains all the functions to interact with user endpoints.
+    """
+
+    def __init__(self):
+        pass
+    
+    @staticmethod
+    def create_users(quay_url, usernames):
+        """
+        Create a series of Users within the test organization.
+
+        :param usernames: list of usernames
+        :return: None
+        """
+        print_header("Running: Create Users", quantity=len(usernames))
+        path = '/api/v1/superuser/users'
+        url = quay_url + path
+        reqs = []
+        for name in usernames:
+            body = {'username': name, 'email': '%s@example.com' % name}
+            request = {
+                'body': body,
+                'url': url,
+                'method': 'POST'
+            }
+            reqs.append(request)
+        target_name = "'POST %s'" % path
+        Attacker().run_vegeta('create_users', reqs, target_name=target_name)
+
+    @staticmethod
+    def update_passwords(quay_url, usernames, password):
```

Comment:
something to note here: This function will work only on a standalone quay deployment and not on quay.io because superuser is disabled in quay.io

Reply:
Acknowledged. I am keeping it same for now. Will replace them as below once we get to `quay.io` in our testing.
* `POST /api/v1/superuser/users` -> `POST /api/v1/user/` for creating users, which is working.
* But `PUT /api/v1/superuser/users` -> `PUT /api/v1/user/` is not working (Getting 500 error response), may be due to users not having admin roles while being created using `/api/v1/user` endpoint. Ref: [code](https://github.com/quay/quay/blob/master/endpoints/v1/index.py#L167).

So is there a way to do a PUT operation on `/api/v1/user` endpoint or is there an endpoint exposed for update user role to `admin` (Couldn't find one from the list of exposed endpoints)? Because we are using this PUT operation to update user passwords. Please let me know your thoughts on this. Thanks!